### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [marizaj/f1-3-c2p1-colmar-academy#1]
Alt text should be used for every image.  This helps the visually impaired know what each picture is when their screen readers scan the page.
